eng/service.proj

```diff
@@ -63,4 +63,12 @@
     <Message Text="Final Build References:" Importance="high"/>
     <Message Text="  %(ProjectReference.Identity)" Importance="high"/>
   </Target>
+
+  <Target Name="GetAllPackageInfo">
```

Comment:
Usually, we reuse the same target name here as inside individual projects.

Reply:
That's fair I can make that change.